packages/gatsby-theme/src/components/Image/Image.stories.mdx

```diff
@@ -50,4 +56,16 @@ Use to insert image
       caption="A caption with a <a href='/'>link</a>"
     />
   </Story>
-</Preview>
+</Canvas>
+
+### With Gatsby Image
+
+<Canvas>
+  <Story name='Gatsby Img'>
```

Comment:
Maybe we could make this story a preview of the base64 image that gatsby provides, something like: `Gatsby Img Base64` we'll also need to change the data.json file containing the gatsby image to prevent console errors. 

Currently, I'm getting a 404 with the current source of the `jpg` of the image

Reply:
thank you so much @edvicaty, I'm going to correct this